packages/bpk-component-chip/src/BpkDismissibleChip.tsx

```diff
@@ -16,31 +16,34 @@
  * limitations under the License.
  */
 
-/* @flow strict */
-
 /*
 The dismissible chip component is just a selectable chip that's
 been hard coded to have disabled={false}, selected and a trailing
 accessory view of a close icon.
 */
 
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
 import { cssModules } from '../../bpk-react-utils';
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
 import CloseCircleIconSm from '../../bpk-component-icon/sm/close-circle';
 
 import BpkSelectableChip from './BpkSelectableChip';
-import {
-  COMMON_PROP_TYPES,
-  COMMON_DEFAULT_PROPS,
-  type CommonProps,
-} from './commonTypes';
+import type { CommonProps } from './commonTypes';
+import { CHIP_TYPES, COMMON_DEFAULT_PROPS } from './commonTypes';
 import STYLES from './BpkSelectableChip.module.scss';
 
-export type Props = CommonProps;
+export interface Props extends CommonProps {}
 
 const getClassName = cssModules(STYLES);
 
-const BpkDismissibleChip = (props: Props) => {
-  const { className, type, ...rest } = props;
+const BpkDismissibleChip = ({
+  className,
+  type = CHIP_TYPES.default,
+  leadingAccessoryView = null,
+  ...rest
+}: Props) => {
+  console.log(type);
```

Comment:
Should this be here?

Reply:
oh God.. good catch! left it accidentally while I was checking something